app/templates/providers/show/prefixes/index.hbs

```diff
@@ -1,49 +1,5 @@
 <div class="row">
   <div class="col-md-3">
-    {{#if
```

Comment:
@svogt0511 I think this was not required to be deleted in the product Spec

Reply:
Any chance we can add this back in? It's a little confusing that you can remove a prefix from a member but not add to a member. Or at very least we need it for staff in case something funny goes on.

Reply:
I will look at this today.  It should be doable today.